CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# TBD
+
+- Add appium options to the base capabilities dictionary [606](https://github.com/bugsnag/maze-runner/pull/606)
```

Comment:
Not sure if we'd consider this a fix or an enhancement?  Maybe just a fix and release it at 8.12.1?